session/src/test/java/org/apache/iotdb/session/pool/SessionPoolTest.java

```diff
@@ -163,7 +163,7 @@ private void correctQuery(SessionPool pool) {
 
   @Test
   public void tryIfTheServerIsRestart() {
-    SessionPool pool = new SessionPool("127.0.0.1", 6667, "root", "root", 3, 1, 6000, false);
+    SessionPool pool = new SessionPool("127.0.0.1", 6667, "root", "root", 3, 1, 6000, false, null);
```

Comment:
So we have to fill a `null` to the constructor of SessionPool if we don't want to set timezone, right? 
I think we'd better don't change these tests and add an extra test of setting timezone.

Reply:
Not Exactly. We can use a constructor which doesn't need a timezone parameter instead.